src/main/java/racinggame/car/RacingGame.java

```diff
@@ -4,27 +4,41 @@
 import racinggame.strategy.MovingStrategy;
 import racinggame.ui.ResultView;
 
+import java.util.ArrayList;
+import java.util.List;
+
 public class RacingGame {
 
+    private static final String CAR_NAME_SPLITTER = ",";
     private final Cars cars;
     private final Count num;
 
-    private RacingGame(Cars cars, Count num) {
-        this.cars = cars;
+    private RacingGame(String carNames, Count num) {
+        this.cars = initCars(carNames);
         this.num = num;
     }
 
-    public static RacingGame of(int carNumber, int countNumber) {
-        return new RacingGame(Cars.manyOf(Count.of(carNumber)), Count.of(countNumber));
+    public static RacingGame of(String carNames, int countNumber) {
+        return new RacingGame(carNames, Count.of(countNumber));
     }
 
     public ResultView play() {
         MovingStrategy strategy = MovingFactory.movingStrategy();
-        int count = num.getCount();
+        int count = num.initValue();
         for (int i = 0; i < count; i++) {
             cars.race(strategy);
         }
 
         return new ResultView(cars, count);
     }
+
+    private Cars initCars(String carNames) {
+        String[] names = carNames.split(CAR_NAME_SPLITTER);
+        List<Car> list = new ArrayList<>();
+        for (String name : names) {
+            list.add(new Car(CarName.of(name)));
+        }
+
+        return Cars.manyOf(list);
```

Comment:
람다식 활용을 아주 잘해주셨으니 
이 부분도 한번 람다식으로 표현 해 볼 수 있을 것 같습니다 😄 
```suggestion
        return Arrays.stream(carNames.split(CAR_NAME_SPLITTER))
                .map(name -> new Car(CarName.of(name)))
                .collect(Collectors
                        .collectingAndThen(Collectors.toList(), 
                                Cars::manyOf));
```

이 부분도 참고만 해주시면 감사하겠습니다 🙇

Reply:
적용해 보겠습니다!